lib/cpu_util.ex

```diff
@@ -57,16 +64,38 @@ defmodule CpuUtil do
   ]a
 
   @doc """
-  Get the current OS PID
+  Get the current OS PID.
+
+  ## Examples
+
+      iex> CpuUtil.getpid() |> is_integer()
+      true
   """
   @spec getpid() :: integer()
   def getpid, do: List.to_integer(:os.getpid())
 
   @doc """
   Get the number of CPU Cores.
+
+  Deprecated! Use `CpuUtl.core_count/0 instead.
   """
   @spec num_cores() :: {:ok, integer()} | :error
   def num_cores do
+    Logger.warn("deprecated. use core_count/0 instead")
```

Comment:
It's a brand new project. Still at 0.1.x. Just delete the thing from your other project and delete this completely.